packages/p2p-v2/src/components/Input/Input.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import clsx from 'clsx';
+import { useFormContext, UseFormReturn } from 'react-hook-form';
+import { Text } from '@deriv-com/ui/dist/components/Text';
+import { useDevice } from '../../hooks';
+import './Input.scss';
+
+type TInputProps = {
+    errorMessage?: string;
+    hasError?: boolean;
+    name: string;
+    options?: Parameters<UseFormReturn['register']>[1];
+    placeholder?: string;
+};
+
+const Input = ({ errorMessage, hasError, name, options, placeholder }: TInputProps) => {
+    const { register } = useFormContext();
```

Comment:
`@deriv-com/ui` has Input right?

Reply:
So for now i created this, but idk what the Input from UI will be like, since for p2p-v2 we wanted to use react-hook-form